activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -106,12 +106,6 @@ def loaded?
       #   #      #<Pet id: 2, name: "Spook", person_id: 1>,
       #   #      #<Pet id: 3, name: "Choo-Choo", person_id: 1>
       #   #    ]
-      #
-      #   person.pets.select(:name) { |pet| pet.name =~ /oo/ }
-      #   # => [
-      #   #      #<Pet id: 2, name: "Spook">,
-      #   #      #<Pet id: 3, name: "Choo-Choo">
-      #   #    ]
```

Comment:
Updated the documentation as well to remove the example showing arguments and block passed at the same time to `select` because this behavior is deprecated now.